frontend/www/js/omegaup/components/qualitynomination/Details.vue

```diff
@@ -1,130 +1,149 @@
+            <div class="col-sm-8">
+              <pre class="border rounded bg-light">{{ this.contents }}</pre>
+            </div>
           </div>
-        </div>
-        <div class="row">
-          <div class="col-sm-3">
-            <strong>{{ T.banProblemFormQuestion }}</strong>
-            <span
-              aria-hidden="true"
-              class="glyphicon glyphicon-info-sign"
-              data-placement="top"
-              data-toggle="tooltip"
-              v-bind:title="T.banProblemFormComments"
-            ></span>
+          <div class="row">
+            <div class="col-sm-3">
+              <strong>{{ T.banProblemFormQuestion }}</strong>
+              <span
+                aria-hidden="true"
+                class="glyphicon glyphicon-info-sign"
+                data-placement="top"
+                data-toggle="tooltip"
+                v-bind:title="T.banProblemFormComments"
+              ></span>
+            </div>
+            <div
+              class="col-sm-8"
+              v-bind:class="{
+                'has-error': !rationale,
+                'has-success': rationale,
+              }"
+            >
+              <textarea
+                class="form-control"
+                name="rationale"
+                type="text"
+                v-model="rationale"
+              ></textarea>
+            </div>
           </div>
           <div
-            class="col-sm-8"
-            v-bind:class="{ 'has-error': !rationale, 'has-success': rationale }"
+            class="row"
+            v-if="this.nomination == 'demotion' &amp;&amp; this.reviewer == true"
           >
-            <textarea
-              class="form-control"
-              name="rationale"
-              type="text"
-              v-model="rationale"
-            ></textarea>
-          </div>
-        </div>
-        <div
-          class="row"
-          v-if="this.nomination == 'demotion' &amp;&amp; this.reviewer == true"
-        >
-          <div class="col-sm-3">
-            <strong>{{ T.wordsVerdict }}</strong>
-          </div>
-          <div class="col-sm-8">
-            <button
-              class="btn btn-danger"
-              v-bind:disabled="!rationale"
-              v-on:click="markResolution('banned')"
-            >
-              {{ T.wordsBanProblem }}
-            </button>
-            <button
-              class="btn btn-success"
-              v-bind:disabled="!rationale"
-              v-on:click="markResolution('resolved')"
-            >
-              {{ T.wordsKeepProblem }}
-            </button>
-            <button
-              class="btn btn-warning"
-              v-bind:disabled="!rationale"
-              v-on:click="markResolution('warning')"
-            >
-              {{ T.wordsWarningProblem }}
-            </button>
+            <div class="col-sm-3">
+              <strong>{{ T.wordsVerdict }}</strong>
+            </div>
+            <div class="col-sm-8 button-group">
```

Comment:
```suggestion
            <div class="col-sm-8 text-center">
```

https://getbootstrap.com/docs/4.4/utilities/text/#text-alignment